client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/connected/RemoteModule.java

```diff
@@ -25,4 +25,6 @@
 
   String getName();
 
+  boolean isTopModule();
```

Comment:
We usually name this "isRoot()" in SonarQube world so maybe we should do the same in SLCORE.